stdlib/public/Concurrency/AsyncLet.cpp

```diff
@@ -165,7 +164,13 @@ static void swift_asyncLet_endImpl(AsyncLet *alet) {
   // TODO: we need to implicitly await either before the end or here somehow.
 
   // and finally, release the task and free the async-let
```

Comment:
```suggestion
  // and finally deallocate the async-let
```

no releasing anymore (yay!)